src/DatabaseDriver/Drivers/Laudis/Relation.php

```diff
@@ -0,0 +1,323 @@
+<?php
+
+namespace Vinelab\NeoEloquent\DatabaseDriver\Drivers\Laudis;
+
+use Laudis\Neo4j\Client;
+use Laudis\Neo4j\Databags\Statement;
+use Vinelab\NeoEloquent\DatabaseDriver\Interfaces\RelationInterface;
+
+class Relation implements RelationInterface
+{
+    /**
+     * @var Client
+     */
+    protected $client;
+    protected $id;
+    protected $properties = [];
+    protected $type;
+    protected $start;
+    protected $end;
+    protected $direction;
+
+    public function __construct($client)
+    {
+        $this->client = $client;
+    }
+
+    public function hasId(): bool
+    {
+        return $this->id !== null;
+    }
+
+    protected function runStatement($cypher, $binding)
+    {
+        $statement = new Statement($cypher, $binding);
+
+        return $this->client->runStatement($statement);
+    }
+
+    protected function compileCreateRelationship(): string
+    {
+        $cypher = "MATCH (a), (b)
+            WHERE id(a) = {$this->start->getId()}
```

Comment:
If possible then it would be better to parametrize the ID's here. It makes it easier for Neo4J to cache the queries.

Reply:
do you mean to add them as a parameter, right? I was a little scared about this, it could match with the properties that the developer send, then there is a mess there. I could create a dynamic variable and check that the variable name is not in the parameters, but then I guess no caching?